### PR TITLE
add date range options

### PR DESCRIPTION
**Pull request by Abdull-Rauf**
stocks-before
stocks-now

### CODE REVIEW

File: src/components/DateRange.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Button from "./styles/Button.styled";
+import Container from "./styles/Container.styled";
+
+const DateRange = ({ handleClick }: any) => {
+  const dateRangeData: any = [
```
Comment (Farhan-Nazir) [inline src/components/DateRange.tsx]: no any type please.

Comment (Farhan-Nazir) [inline src/components/DateRange.tsx]: I will suggest you to create a `config` file and add ranges in that file.

File: src/components/StocksChart.tsx [hunk 2]
```diff
@@ -31,14 +33,19 @@ const StocksChart: FC<StocksChartProps> = ({ stocksData, symbol }) => {
 
     const labels = formattedData.map((obj: any) => obj.date);
     const data = formattedData.map((obj: any) => obj.priceData["2. high"]);
```
Comment (Farhan-Nazir) [inline src/components/StocksChart.tsx]: please try to add type instead of `any`

**Review by Farhan-Nazir — CHANGES_REQUESTED** [review]

File: src/components/DateRange.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { FC } from "react";
+import Button from "./styles/Button.styled";
+import Container from "./styles/Container.styled";
+import { dateRangeData } from "../config/config";
+import { HandleClickType } from "types";
+const DateRange: FC<HandleClickType> = ({ handleClick }) => {
+  return (
+    <Container>
+      {dateRangeData.map((range: any, i: number) => {
```
Comment (Farhan-Nazir) [inline src/components/DateRange.tsx]: Set type please

File: src/types.d.ts [hunk 4]
```diff
@@ -4,8 +4,17 @@ declare module "types" {
   };
   export type StocksResponseType = {
     [key: string]: {
-      [key: string]: string;
+      [key: string]: any;
```
Comment (Farhan-Nazir) [inline src/types.d.ts]: Set type please

**Review by Farhan-Nazir — CHANGES_REQUESTED** [review]

File: src/components/DateRange.tsx (hunk 3 above)
Comment (Abdull-Rauf) [reply src/components/DateRange.tsx]: added types

File: src/types.d.ts (hunk 4 above)
Comment (Abdull-Rauf) [reply src/types.d.ts]: types added

**Review by Farhan-Nazir — APPROVED** [review]

**Pull request merged**